src/widgets/medications/medication-level-two.component.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { match } from "react-router";
+import SummaryCard from "../cards/summary-card.component";
+import { fetchPatientMedications } from "./medications.resource";
+import styles from "./medication-level-two.css";
+import { formatDate } from "../heightandweight/heightandweight-helper";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import { useCurrentPatient } from "@openmrs/esm-api";
+import SummaryCardFooter from "../cards/summary-card-footer.component";
+import { useTranslation } from "react-i18next";
+
+export default function MedicationLevelTwo(props: MedicationsOverviewProps) {
+  const [patientMedications, setPatientMedications] = React.useState(null);
+  const [
+    isLoadingPatient,
+    patient,
+    patientUuid,
+    patientErr
+  ] = useCurrentPatient();
+  const pastMedication = false;
+
+  const { t } = useTranslation();
+
+  React.useEffect(() => {
+    const subscription = fetchPatientMedications(patientUuid).subscribe(
+      Medications => setPatientMedications(Medications),
+      createErrorHandler()
+    );
+    return () => subscription.unsubscribe();
+  }, [patientUuid]);
+
+  function displayMedication() {
+    return (
+      <React.Fragment>
+        <SummaryCard
```

Comment:
What is displayed if there are no medications, add a ternary operator like below to show when the patient has or doesn't have medications.
```suggestion
        {patientMedications && patientMedications.length ? <SummaryCard
```